lib/wings/services/file_node_builder.rb

```diff
@@ -33,16 +33,13 @@ def create(io_wrapper:, node:, file_set:)
     end
 
     def attach_file_node(node:, file_set:)
-      saved_node = file_set.is_a?(::Valkyrie::Resource) ? attach_file_node_to_valkyrie_file_set(node, file_set) : node
-
-      # note the returned saved_node does not yet contain the characterization done in the async job
-      CharacterizeJob.perform_later(saved_node.file_identifiers.first.to_s) # TODO: What id is the correct one for the file? Check where this is called outside of wings.
```

Comment:
Removing this here.  It wasn't working and is now done in the file_actor.  The file_actor performs this when not working with Valkyrie, so it seems like a consistent place to do it when working with Valkyrie.